README.md

```diff
@@ -2366,6 +2413,7 @@ IEEE Robotics and Automation Letters (RA-L), 2021. [Dataset](http://rpg.ifi.uzh.
 - [Zhu et al., RAL2018](#Zhu18mvsec): *MVSEC The Multi Vehicle Stereo Event Camera Dataset*.
 - [Almatrafi et al. PAMI 2020](#Almatrafi20arxiv): *Distance Surface for Event-Based Optical Flow*. [DVSMOTION20 Dataset](https://sites.google.com/a/udayton.edu/issl/software/dataset?authuser=0)
 - [EDFLOW21 Event Driven Optical Flow Camera dataset](https://sites.google.com/view/edflow21/home) associated with the paper [EDFLOW: Event Driven Optical Flow Camera with Keypoint Detection and Adaptive Block Matching](#LiuDelbruck2022tcsvt).
+- [Gehrig et al., RAL21](#MGehrig21ral): *DSEC: A Stereo Event Camera Dataset for Driving Scenarios*.  
```

Comment:
Try to be consistent with the notation. One is "Mahlknecht et al., RAL 2022" and the other one is " ... RAL21".
Try to be consistent with the other entries in the file.